### DISCUSSION TITLE
How to insert dynamic text in Onlyoffice from an external trigger?

### DISCUSSION THREAD

**Discussion by ulengoots** [ONLYOFFICE/DocumentServer#2991] [Q&A]
Hello,

Currently, ONLYOFFICE provides the insertImage function, which allows me to dynamically insert an image into the  Onlyoffice document from an external trigger. Is there a similar function available for inserting text in the same way as insertImage? Thank you!

**Comment by Rita-Bubnova** [comment]
@regent112 Take a look, please.
